src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,10 +24,31 @@
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
+  LogitechGamingPad drivePad;
+
+  DriveTrain driveTrain;
+  Shooter shooter;
+  Intake intake;
+
+
+  JoystickButton driveA;
+  JoystickButton driveB;
+  JoystickButton driveX;
+  JoystickButton driveLeftBumper; 
+  JoystickButton driveRightBumper; 
+
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     // Configure the button bindings
+    
+    drivePad = new LogitechGamingPad(0);
+
+    driveTrain = new DriveTrain();
+    shooter = new Shooter();
+    intake = new Intake();
```

Comment:
```suggestion
    intake = new Intake();
```
Similar reasoning as to previous comment dealing w/ the initialization of the subsystems.